data/json/items/migration.json

```
@@ -1268,6 +1268,21 @@
     "type": "MIGRATION",
     "replace": "condiment_bottle_sealed"
   },
+  {
+    "id": "acid",
+    "type": "MIGRATION",
+    "replace": "chem_sulphuric_acid"
+  },
+  {
+    "id": "water_acid",
+    "type": "MIGRATION",
+    "replace": "chem_sulphuric_acid"
+  },
+  {
+    "id": "water_acid_weak",
+    "type": "MIGRATION",
+    "replace": "chem_sulphuric_acid"
```

```suggestion
    "replace": "chem_acetic_acid"
```
May as well keep it a relatively weak acid